RedirectApplication/RedirectApplication/Json/Json.cs

```diff
@@ -0,0 +1,167 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace RedirectApplication.Json
+{
+    public class JsonWorking
+    {
+        public void Main(string tree)
+        {
+            var reader = new JsonTextReader(new StringReader(tree));
+            PostJson content = JsonSerializer.CreateDefault().Deserialize<PostJson>(reader);
```

Comment:
Lets be consistent during variable declaration. If you're using `var`, keep using it everywhere.